Networking/Networking/Remote/InboxNotesRemote.swift

```diff
@@ -108,15 +113,40 @@ public final class InboxNotesRemote: Remote, InboxNotesRemoteProtocol {
     ///
     /// - Parameters:
     ///     - siteID: The site for which we'll dismiss all the InboxNotes.
+    ///     - pageNumber: The page number of the Inbox Notes list to be fetched.
+    ///     - pageSize: The maximum number of Inbox Notes to be fetched for the current page.
+    ///     - orderBy: The type of sorting that the Inbox Notes list will follow.
+    ///     - type: The array of Inbox Notes Types.
+    ///     - status: The array of Inbox Notes with a specific array of status that will be fetched.
     ///     - completion: Closure to be executed upon completion.
     ///
     public func dismissAllInboxNotes(for siteID: Int64,
+                                     pageNumber: Int = Default.pageNumber,
+                                     pageSize: Int = Default.pageSize,
+                                     orderBy: InboxNotesRemote.OrderBy = .date,
+                                     type: [InboxNotesRemote.NoteType]? = nil,
+                                     status: [InboxNotesRemote.Status]? = nil,
```

Comment:
I'm just curious: why do we need default values for the parameters here (as well as in Yosemite layer)? As it looks like we're inputting the values on the UI layer, and it makes more sense that the lower layers don't need to know about the business logic.